infrastructure/terraform/modules/dpl-platform-environment/variables.tf

```diff
@@ -9,12 +9,6 @@ variable "control_plane_version" {
   type        = string
 }
 
-variable "cluster_upgrade_channel" {
-  description = "Which channel to use for automatic cluster upgrades. Valid values are 'stable', 'rapid', 'patch' and 'node-image'."
-  type        = string
-  default     = "patch"
-}
-
```

Comment:
I am a complete terraform novice.
But isn't there supposed to be a `control_plane_version` variable block in this file?

Reply:
Yup, it's there - in line 7 😉